python/ray/scripts/scripts.py

```diff
@@ -80,14 +86,17 @@ def cli():
 @click.option("--autoscaling-config", required=False, type=str,
               help="the file that contains the autoscaling config")
 def start(node_ip_address, redis_address, redis_port, num_redis_shards,
-          redis_max_clients, object_manager_port, num_workers, num_cpus,
-          num_gpus, resources, head, no_ui, block, plasma_directory,
-          huge_pages, autoscaling_config):
-    # Note that we redirect stdout and stderr to /dev/null because otherwise
-    # attempts to print may cause exceptions if a process is started inside of
-    # an SSH connection and the SSH connection dies. TODO(rkn): This is a
-    # temporary fix. We should actually redirect stdout and stderr to Redis in
-    # some way.
+          redis_max_clients, object_manager_port, redis_shard_ports,
```

Comment:
Why not group the redis arguments together?

Reply:
fixed